connection.go

```diff
@@ -301,7 +314,37 @@ func (sc *snowflakeConn) ExecContext(ctx context.Context, query string, args []d
 	return driver.ResultNoRows, nil
 }
 
+// QueryContext when there is
 func (sc *snowflakeConn) QueryContext(ctx context.Context, query string, args []driver.NamedValue) (driver.Rows, error) {
+	qid, _ := getResumeQueryID(ctx)
+	if qid == "" {
+		return sc.queryContextInternal(ctx, query, args)
+	}
+
+	// so there is a queryID in context for which we want to fetch the result
+	pattern := `[\w\-_]+`
```

Comment:
should be regexp.MustCompile() and made global

Reply:
addressed